Dockerfile

```diff
@@ -0,0 +1,36 @@
+##
+## Step 1 - Get dependencies
+##
+FROM golang:1.16-alpine as otp-builder
+
+WORKDIR /build
+
+RUN apk update --no-cache && \
+    apk upgrade --no-cache && \
+    apk add --no-cache \
+    make \
+    build-base
+
```

Comment:
RUN go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest

Reply:
Not anymore.